v2/sourcedb-to-spanner/src/main/java/com/google/cloud/teleport/v2/source/reader/io/row/SourceRow.java

```diff
@@ -29,6 +30,20 @@
 @AutoValue
 public abstract class SourceRow implements Serializable {
 
+  /**
+   * Get the namespace of the database for the row.
+   *
+   * @return namespace
+   */
+  public abstract String namespace();
```

Comment:
Please mark this nullable.

Reply:
No longer needed since we using SourceSchemaRef now.